src/tests/SimTest.cpp

```diff
@@ -124,8 +164,116 @@ void SimTest::getPinholeCameraRGBAObservation() {
           Mn::PixelFormat::RGBA8Unorm,
           {pinholeCameraSpec->resolution[0], pinholeCameraSpec->resolution[1]},
           observation.buffer->data}),
-      Cr::Utility::Directory::join(TEST_DIR, "SimTestExpected.png"),
-      (Mn::DebugTools::CompareImageToFile{15.0f, 0.17f}));
+      Cr::Utility::Directory::join(screenshotDir, groundTruthImageFile),
```

Comment:
I added the `TEST_DIR` variable to `src/tests/configure.h.cmake` in #496, so if it's not needed anymore, can you remove it again? Thanks!